agent/rpm/Makefile

```diff
@@ -23,13 +23,30 @@ CHROOTS =
 # These targets build the binary RPM for the specified distro version; the "ci"
 # target builds them all.
 RHEL_RPMS = rhel-9-rpm rhel-8-rpm rhel-7-rpm
-CENTOS_RPMS = centos-9-rpm centos-8-rpm
-FEDORA_RPMS = fedora-34-rpm fedora-35-rpm fedora-36-rpm
+CENTOS_RPMS = centos-9-rpm centos-8-rpm centos-7-rpm
+FEDORA_RPMS = fedora-36-rpm fedora-35-rpm
 ALL_RPMS = ${RHEL_RPMS} ${CENTOS_RPMS} ${FEDORA_RPMS}
 
 component = agent
 subcomps = agent
 
+# Generate a sequence number to be used for the RPMs:  by default, the number
```

Comment:
I don't understand why we need to have this code copied in both the agent and server Makefile.  Can you explain that a bit more?

Reply:
Yeah, the duplication is regrettable.  I could move the guts of the expression to a Make user-function with a common definition, but the evaluation of it has to happen here (and, therefore, also in the Server makefile).

The requirement is that we generate a value for `seqno` exactly once per build.  Note that `make -C agent/rpm ci` invokes `make` repeatedly on the `utils/rpm.mk` makefile, once for each distro; so if we were to place the generation of `seqno` there (where it use to be), then each distro would receive a different `seqno` value, which is what I was trying to fix.  (The `ci` target is set up so that if the top-level `make` is invoked with the `-j` switch, then we each distro is built in parallel with the others.)

Reply:
Why don't a simple `include seq.mk` just work?

Reply:
It probably would work, but I don't really like that solution.  I'd like to avoid a proliferation of little include files.  I'm concerned about include files which have to be included at a particular point in the file which includes them.  And, somehow, it feels funny to include a file in order to get the sequence number value (it doesn't bother me to include a file to get "constant" values or functional definitions, but something about pulling this from a file does bother me, perhaps because its "actively" calculated).

I'll see if I can roll the guts of the expression into a function and put it in `utils.mk` and have the Agent and Server just invoke the common definition.